scripts/zephyr_module.py

```diff
@@ -758,6 +758,9 @@ def parse_modules(zephyr_base, manifest=None, west_projs=None, modules=None,
 
     if extra_modules is None:
         extra_modules = []
+        for var in ['EXTRA_ZEPHYR_MODULES', 'ZEPHYR_EXTRA_MODULES']:
+            if var in os.environ:
```

Comment:
are you sure CMake exports them as environment variables? because those can be CMake-only variables I think?

Reply:
No, calling from cmake they will be passed as arguments, so `extra_modules` won't be `None`.